CHANGES.rst

```diff
@@ -2,6 +2,8 @@
 --------
 
 - Introduces pypeit_parse_calib_id script
+- Pypeit can now read the list of arc lamps from the header and use it for
+  wavelength calibration.
```

Comment:
Almost not worth mentioning, but I would specify that this is currently for DEIMOS only.